mlflow/pyfunc/__init__.py

```diff
@@ -627,7 +627,6 @@ def _load_model_or_server(model_uri: str, env_manager: str):
         local_path,
         env_manager=env_manager,
         install_mlflow=os.environ.get("MLFLOW_HOME") is not None,
-        create_env_root_dir=True,
```

Comment:
Do we still need this arg in `mlflow.pyfunc.backend.PyFuncBackend` if it's always defaulted to `False` now?

Reply:
Yes. it's used here: https://github.com/mlflow/mlflow/blob/4978967198ff13a3ef2f7aabaf517f4dd636b159/mlflow/pyfunc/__init__.py#L630